### PR TITLE
tlaplus-toolbox: Add version 1.7.1

### PR DESCRIPTION
**Pull request by goto1134**
Add TLA+ Toolbox

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by goto1134** [comment]
@charlesroper, I need a human here. 

I checked the autoupdate and it works. I assume that the issue in the CI is the following:

All the releases in TLA+ repository contain the windows archives, except the latest [1.7.2 version](https://github.com/tlaplus/tlaplus/releases/tag/v1.7.2). It is a hotfix version of a single jar file and does not include the IDE itself. The release page says to go to the previous release page to download the artifacts. 

If you pass the pre-release version 1.8.0 as the target to update scoop, the update will succeed:
```
➜ .\bin\checkver.ps1 tla-plus-toolbox -u -v 1.8.0
tla-plus-toolbox: 1.8.0 (scoop version is 1.7.1) autoupdate available
Autoupdating tla-plus-toolbox
DEBUG[1672431748,54478] [$updatedProperties] = [url hash] -> C:\Users\goto1134\scoop\apps\scoop\current\lib\autoupdate.ps1:442:5
DEBUG[1672431748,56144] $substitutions (hashtable) -> C:\Users\goto1134\scoop\apps\scoop\current\lib\autoupdate.ps1:180:5
DEBUG[1672431748,56144] $substitutions.$basename                      TLAToolbox-1.8.0-win32.win32.x86_64.zip                                                                                                                           

DEBUG[1672431748,56144] $substitutions.$preReleaseVersion             1.8.0                                                                                                                                                             

DEBUG[1672431748,56144] $substitutions.$baseurl                       https://github.com/tlaplus/tlaplus/releases/download/v1.8.0                                                                                                       

DEBUG[1672431748,56144] $substitutions.$url                           https://github.com/tlaplus/tlaplus/releases/download/v1.8.0/TLAToolbox-1.8.0-win32.win32.x86_64.zip                                                               

DEBUG[1672431748,56144] $substitutions.$dashVersion                   1-8-0                                                                                                                                                             

DEBUG[1672431748,56144] $substitutions.$underscoreVersion             1_8_0                                                                                                                                                             

DEBUG[1672431748,56144] $substitutions.$urlNoExt                      https://github.com/tlaplus/tlaplus/releases/download/v1.8.0/TLAToolbox-1.8.0-win32.win32.x86_64                                                                   

DEBUG[1672431748,56144] $substitutions.$minorVersion                  8                                                                                                                                                                 

DEBUG[1672431748,56144] $substitutions.$patchVersion                  0                                                                                                                                                                 

DEBUG[1672431748,56144] $substitutions.$basenameNoExt                 TLAToolbox-1.8.0-win32.win32.x86_64                                                                                                                               

DEBUG[1672431748,56144] $substitutions.$dotVersion                    1.8.0                                                                                                                                                             

DEBUG[1672431748,56144] $substitutions.$matchTail                                                                                                                                                                                       

DEBUG[1672431748,56144] $substitutions.$buildVersion                                                                                                                                                                                    

DEBUG[1672431748,56144] $substitutions.$majorVersion                  1                                                                                                                                                                 

DEBUG[1672431748,56144] $substitutions.$cleanVersion                  180                                                                                                                                                               

DEBUG[1672431748,56144] $substitutions.$matchHead                     1.8.0                                                                                                                                                             

DEBUG[1672431748,56144] $substitutions.$version                       1.8.0                                                                                                                                                             

DEBUG[1672431748,62374] $hashfile_url = https://github.com/tlaplus/tlaplus/releases/tag/v1.8.0 -> C:\Users\goto1134\scoop\apps\scoop\current\lib\autoupdate.ps1:183:5
Searching hash for TLAToolbox-1.8.0-win32.win32.x86_64.zip in https://github.com/tlaplus/tlaplus/releases/tag/v1.8.0
DEBUG[1672431749,19446] $regex = <td>([a-fA-F0-9]{40})</td>\s+<td>TLAToolbox-1\.8\.0-win32\.win32\.x86_64\.zip</td> -> C:\Users\goto1134\scoop\apps\scoop\current\lib\autoupdate.ps1:53:5
Found: sha1:86520aa6a755dccf430b9e12830efb5b97f74442 using Extract Mode
Writing updated tla-plus-toolbox manifest
```

I assume, that is a one-time issue and after the release of 1.8.0 it will not happen again. Therefore I think, you might accept the PR despite the failing CI checks.

**Comment by goto1134** [comment]
@HUMORCE, can you help me with the problem explained here? https://github.com/ScoopInstaller/Extras/pull/10141#issuecomment-1367995127

File: bucket/tla-plus-toolbox.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+    "suggest": {
+        "pretty printer": [
+            "miktex",
+            "latex"
+        ]
+    },
+    "checkver": {
+        "github": "https://github.com/tlaplus/tlaplus"
+    },
```
Comment (HUMORCE) [inline bucket/tla-plus-toolbox.json:30]:
```suggestion
    "checkver": {
        "github": "https://github.com/tlaplus/tlaplus",
        "regex": "TLAToolbox-([\\d.]+)-win32.win32.x86_64.zip"
    },
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]
rename manifest to `tlaplus`

**Comment by HUMORCE** [comment]
maybe `tlaplus-toolbox`?

```
'extras' bucket:
clavier-plus (11.3.0)
dismplusplus (10.1.1002.1)
explorerplusplus (1.4.0-beta-2)
nicotine-plus (3.2.8)
notepadplusplus (8.4.8)
office-tool-plus (10.0.4.7)
openwithplusplus (4.0.0.0)
strokesplus (2.8.6.4)
strokesplus.net (0.5.7.2)
tableplus (5.0.4.214)
```

**Comment by goto1134** [comment]
> maybe `tlaplus-toolbox`?

Sure

**Comment by goto1134** [comment]
@HUMORCE , I applied your fix

File: bucket/tla-plus-toolbox.json (hunk 1 above)
Comment (goto1134) [reply bucket/tla-plus-toolbox.json]: Now it fails the checkver :(

Comment (HUMORCE) [reply bucket/tla-plus-toolbox.json]: Expected, the latest release does not contain win binary.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**